packages/cxl-ui/src/components/cxl-light-card.js

```diff
@@ -1,17 +1,44 @@
 /* eslint-disable import/no-extraneous-dependencies */
-import { html, nothing } from 'lit';
+import { html, LitElement, nothing } from 'lit';
 import { unsafeHTML } from 'lit/directives/unsafe-html.js';
-import { customElement, property } from 'lit/decorators.js';
-import '@conversionxl/cxl-lumo-styles';
+import { customElement, property, state } from 'lit/decorators.js';
+
+import '@vaadin/progress-bar';
+
 import cxlLightCardStyles from '../styles/cxl-light-card-css.js';
-import { CXLBaseCardElement } from './cxl-base-card.js';
 
 @customElement('cxl-light-card')
-export class CXLLightCardElement extends CXLBaseCardElement {
+export class CXLLightCardElement extends LitElement {
   static get styles() {
-    return [...super.styles, cxlLightCardStyles];
+    return [cxlLightCardStyles];
   }
 
+  @state() _tagsHasChildren = false;
+
+  @property({ type: String }) id = '';
+
+  @property({ type: String }) theme = '';
+
+  @property({ type: String }) name = '';
+
+  @property({ type: String }) avatar = '';
+
+  @property({ type: String }) time = '';
+
+  @property({ type: String }) instructor = '';
+
+  @property({ type: Number }) progress = 0;
+
+  @property({ type: Number }) lessons = 0;
+
+  @property({ type: Boolean, reflect: true }) new = false;
+
+  @property({ type: Boolean, reflect: true }) portrait = false;
```

Comment:
I think this should be defined by `theme`. @freudFlintstone @anoblet WDYT?

Reply:
It's only used in CSS, so yes, theme would be more consistent with the rest of the components. It is better to have a property only if the component needs to conditionally, and maybe dynamically, render parts of the template depending on that property.